azurelinuxagent/common/cgroupstelemetry.py

```diff
@@ -62,6 +90,14 @@ def _process_cgroup_metric(cgroup_metrics):
             else:
                 processed_extension["memory"] = {"max_mem": CGroupsTelemetry._get_metrics_list(max_memory_usage)}
 
+        if len(memory_usage_per_process) > 0:
```

Comment:
if not needed (for below will take care of empty)

Reply:
neat. Didnt think of it earlier. Thanks.